src/components/MapView/MapView.tsx

```diff
@@ -84,15 +154,27 @@ export let MapView = withStore(
       onDragEnd={(map: MapboxGL.Map) => store.set('mapCenter')(map.getCenter())}
       onZoomEnd={(map: MapboxGL.Map) => store.set('mapZoom')(map.getZoom())}
       zoom={[store.get('mapZoom')]}
+      onClick={() => removePopup(store)}
     >
       {representativePoints.length && <GeoJSONLayer
+        id='representativePoints'
         data={representativePointsToGeoJSON(adequacies)(representativePoints)}
         circlePaint={representativePointCircleStyle}
+        circleOnClick={(point: any) => store.set('representativePointClicked')(point)}
```

Comment:
TypeScript should be able to infer the type for you here, so you can just write `point => store.set(.....`.

With VSCode, just hover over `point` and see if a type tooltip appears.

Reply:
Done